### PR TITLE
Fix/96 Refactor Intervention Detail Test

### PR DESCRIPTION
**Pull request by josgarber6**
## Description
Refactor Intervention Detail Test so it can work in MongoDB

## Related Issue(s)
- #96 

## Checklist
- [x] I have read and followed the project's contributing guidelines.
- [x] I have tested the changes locally.
- [x] I have made corresponding changes to the documentation if necessary.
- [x] My code follows the project's code style guidelines.

## Reviewers
- @alblopnov

### CODE REVIEW

File: tests/api/test_acat_intervention.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
+from uuid import uuid4
+import datetime
+from src.modules.acat.intervention.model import Intervention
```
Comment (jmartinacu) [inline tests/api/test_acat_intervention.py:3]: Esta dependencia no se usa en todo  el archivo, y se tiene que eliminar para no añadir mas issues a codacy.

Comment (josgarber6) [reply tests/api/test_acat_intervention.py:3]: Okey, solving it asap

**Review by alblopnov — APPROVED** [review]
Everything works how its supposed to. Great job!

**Review by gonzcm — APPROVED** [review]
Well done!

**Pull request merged**